cadcutils/cadcutils/net/tests/test_groups_client.py

```diff
@@ -324,7 +324,9 @@ def test_mainapp_get(get_group_mock, basews_mock):
     get_group_mock.reset_mock()
     sys.argv = 'cadc-groups get --cert cert.pem grID1 grID2'.split()
     main_app()
-    get_group_mock.has_calls([call(group_id='grID1'), call(group_id='grID2')])
+    assert 2 == get_group_mock.call_count
+    for ii in get_group_mock.calls:
+        assert ii in [call(group_id='grID1'), call(group_id='grID2')]
```

Comment:
If call order is the reason this change was made, `assert_has_calls` has an arg `any_order` that defaults to `False`.

Reply:
No, it was something different and quite cryptic.